### PR TITLE
Updating branding to 2.2.200

### PR DESCRIPTION
**Pull request by livarcocc**

### CODE REVIEW

File: build/Versions.props [hunk 1]
```diff
@@ -8,8 +8,8 @@
 
   <!-- Repo Version Information -->
   <PropertyGroup>
-    <VersionPrefix>2.2.100</VersionPrefix>
-    <PreReleaseVersionLabel>preview3</PreReleaseVersionLabel>
+    <VersionPrefix>2.2.200</VersionPrefix>
+    <PreReleaseVersionLabel>preview</PreReleaseVersionLabel>
```
Comment (peterhuene) [inline build/Versions.props]: Are we dropping the number from the preview suffix (i.e. not `preview1`)?

Comment (livarcocc) [reply build/Versions.props]: Yes. No more!

**Review by peterhuene — APPROVED** [review]

**Comment by livarcocc** [comment]
@dotnet-bot Test this please

**Comment by livarcocc** [comment]
@dotnet-bot Test OSX10.12 Release

**Comment by livarcocc** [comment]
@dotnet-bot Test OSX10.12 Release

**Pull request merged**